general.postfixTemplates

```diff
@@ -0,0 +1,7 @@
+## general use ##
```

Comment:
can you move these to `misc.postfixTemplates`? My idea was to include general templates in that file 
https://github.com/philipgriggs/go-postfix-templates/blob/master/misc.postfixTemplates

Reply:
done!